package.json

```diff
@@ -50,9 +50,8 @@
     "p-iteration": "^1.1.7"
   },
   "devDependencies": {
-    "bfx-hf-ext-plugin-bitfinex": "^1.0.5",
-    "bfx-hf-models-adapter-lowdb": "^1.0.0",
     "chai": "^4.2.0",
+    "docdash": "^1.2.0",
```

Comment:
whats this requried for?

Reply:
templating the docs .. it throws error when it doesnt find this .. its added in jsdoc config